examples/exalearn/exalearn.ipynb

```diff
@@ -0,0 +1,514 @@
+   "execution_count": 3,
+   "id": "ab21ffb9-ab66-49a9-9357-918dbbeafb2f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Resource Engine started successfully\n",
+      "\n",
+      "Task 'sample_simulation' ready to submit; resolved dependencies: []\n",
+      "Task 'sweep_simulation' ready to submit; resolved dependencies: []\n",
+      "Task 'sample_simulation' ready to submit; resolved dependencies: []\n",
+      "Task 'sample_simulation' ready to submit; resolved dependencies: []\n",
+      "submitting ['sample_simulation', 'sweep_simulation', 'sample_simulation', 'sample_simulation'] for execution\n",
+      "task.000003 is DONE\n",
+      "task.000001 is DONE\n",
+      "task.000002 is DONE\n",
+      "task.000000 is DONE\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sweep_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sweep_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sample_simulation']\n",
+      "Task 'merge_preprocess' ready to submit; resolved dependencies: ['sweep_simulation']\n",
+      "submitting ['merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess', 'merge_preprocess'] for execution\n",
+      "task.000009 is DONE\n",
+      "task.000008 is DONE\n",
+      "task.000006 is DONE\n",
+      "task.000012 is DONE\n",
+      "task.000007 is DONE\n",
+      "task.000004 is DONE\n",
+      "task.000015 is DONE\n",
+      "task.000010 is DONE\n",
+      "task.000005 is DONE\n",
+      "task.000014 is DONE\n",
+      "task.000013 is DONE\n",
+      "task.000011 is DONE\n",
+      "Task 'training' ready to submit; resolved dependencies: []\n",
+      "submitting ['training'] for execution\n",
+      "task.000016 is FAILED\n",
+      "Task 'active_learn' ready to submit; resolved dependencies: ['training']\n",
+      "submitting ['active_learn'] for execution\n",
+      "task.000017 is FAILED\n",
```

Comment:
Same as above.

Reply:
You can check the reason of failure by checking the produced sessions for both client and agent:

1. Client: located in the same dir where you ran the code from and starts with `rose.xxxx`
2. Agent: located under ```radical.pilot.sandbox/rose.xxxx` with the same session id of the client.

You can do `cd rose.xxxx` and then `grep -r "ERROR" *` or `grep -r "Error" *` . LMK if you need more help please.